hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConsts.java

```diff
@@ -56,8 +56,6 @@
 
   public static final String OZONE_ACL_READ = "r";
```

Comment:
We will also need OZONE_ACL_CREATE, OZONE_ACL_READ_ACP, OZONE_ACL_WRITE_ACP as documented in the design spec.

Reply:
yes, we do. Thanks

Reply:
This is already there in form of following constants. 
  public static final String OZONE_ACL_CREATE = "c";
  public static final String OZONE_ACL_READ_ACL = "x";
  public static final String OZONE_ACL_WRITE_ACL = "y";